app/assets/javascripts/main.coffee

```diff
@@ -485,8 +486,8 @@ window.youtubeDurationSec = (key)  ->
   else
     hour = 0
     d2 = duration
-  min = parseInt(d2.split('M')[0])
-  sec = parseInt(d2.split('M')[1])
+  min = parseInt(d2.split('M')[0]) || 0
+  sec = parseInt(d2.split('M')[1]) || 0
```

Comment:
[nits] it better splited value assign to variable before  access each indexed value

Reply:
@kobake can you fix above?

Reply:
@fukajun Thanks for the review, but I'm not sure I understand your suggestion. It would be helpful if you give me an example code.